docs/ops/normalization/GroupNormalization_12.md

```diff
@@ -0,0 +1,79 @@
+# GroupNormalization {#openvino_docs_ops_normalization_GroupNormalization_12}
+
+@sphinxdirective
+
+**Versioned name**: *GroupNormalization-12*
+
+**Category**: *Normalization*
+
+**Short description**: Performs normalization of the input tensor according to the method described in https://arxiv.org/abs/1803.08494
+
+**Detailed description**
+
+The GroupNormalization operation performs the following transformation of the input tensor:
+
+  `y = scale * (x - mean) / sqrt(variance + epsilon) + bias`
+
+The operation is applied per batch, per group of channels. This means that the example input with `N x C x H x W` layout is transformed to the `N x G x C/G x H x W` form. The `scale` and `bias` coefficients are the inputs to the model and need to be specified separately for each group. The `mean` and `variance` are calculated for each group and applied accordingly.
```

Comment:
Currently double ticks are used in the docs.
Reference: https://github.com/openvinotoolkit/openvino/pull/17158

```suggestion
The operation is applied per batch, per group of channels. This means that the example input with ``N x C x H x W`` layout is transformed to the ``N x G x C/G x H x W`` form. The ``scale`` and ``bias`` coefficients are the inputs to the model and need to be specified separately for each group. The ``mean`` and ``variance`` are calculated for each group and applied accordingly.
```